<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameMenu</class>
 <widget class="QWidget" name="GameMenu">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Menu</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="restartButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="recipeButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>393</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
